### PR TITLE
Next

### PR DESCRIPTION
**Pull request by echotyh**

### CODE REVIEW

**Comment by echotyh** [comment]
Change arch/X86/X86MappingInsnOp.inc  arch/X86/X86MappingInsnOp_reduce.inc to fit instruction manual and xed files.

File: arch/X86/X86Mapping.c [hunk 1]
```diff
@@ -3467,13 +3467,16 @@ void op_addAvxBroadcast(MCInst *MI, x86_avx_bcast v)
 #ifndef CAPSTONE_DIET
 // map instruction to its characteristics
 typedef struct insn_op {
-	uint64_t eflags;	// how this instruction update EFLAGS
+	union{
+		uint64_t eflags;	// how this instruction update EFLAGS
+		uint64_t fpuflags;
```
Comment (aquynh) [inline arch/X86/X86Mapping.c]: can you rename this field to fpu_flags, so it is consistent with other fields? (eflags is special, as it is associated with the EFLAGS register of x86)

**Comment by echotyh** [comment]
Rename fpuflags to fpu_flags in arch/X86/X86Mapping.c .

File: arch/X86/X86Mapping.c [hunk 2]
```diff
@@ -3467,13 +3467,13 @@ void op_addAvxBroadcast(MCInst *MI, x86_avx_bcast v)
 #ifndef CAPSTONE_DIET
 // map instruction to its characteristics
 typedef struct insn_op {
-	uint64_t eflags;	// how this instruction update EFLAGS
+	uint64_t flags;	// how this instruction update EFLAGS
```
Comment (aquynh) [inline arch/X86/X86Mapping.c]: the comment on this line should be "how this instruction update EFLAGS (for arithmetic instructions), or FPU FLAGS (for FPU instructions)

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**